geomesa-hbase/geomesa-hbase-datastore/src/test/scala/org/locationtech/geomesa/hbase/data/HBaseDataStoreTest.scala

```diff
@@ -37,6 +37,8 @@ class HBaseDataStoreTest extends Specification with LazyLogging {
   val cluster = new HBaseTestingUtility()
   var connection: Connection = _
 
+  System.setProperty("geomesa.scan.ranges.target", "100")
```

Comment:
should we set this in the maven opts maybe? or at least revert it in the shutdown `step`